clusterloader2/testing/load/config.yaml

```diff
@@ -57,15 +57,27 @@
 
 # BEGIN scheduler-throughput section
 # TODO( https://github.com/kubernetes/perf-tests/issues/1027): Lower the number of "min-pods" once we fix the scheduler throughput measurement.
-{{$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) .Nodes}}
-{{$schedulerThroughputPodsPerDeployment := .Nodes}}
-{{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
+{{$schedulerThroughputReplicasPerNamespace := 0}}
+{{$schedulerThroughputPodsPerDeployment := 0}}
+{{$schedulerThroughputNamespaces := 0}}
+{{if $IS_SMALL_CLUSTER}}
+  {{$schedulerThroughputReplicasPerNamespace = 1}}
+  {{$schedulerThroughputPodsPerDeployment = .Nodes}}
+  {{$schedulerThroughputNamespaces = 1}}
+{{end}}
+{{if and (ge .Nodes 100) (lt .Nodes 1000)}}
+  {{$totalSchedulerThroughputPods := MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS}}
+  {{$schedulerThroughputReplicasPerNamespace = 1}}
+  {{$schedulerThroughputPodsPerDeployment = .Nodes}}
+  {{$schedulerThroughputNamespaces = DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
+{{end}}
+{{if ge .Nodes 1000}}
```

Comment:
Instead of using constant, it should be a customizable value (1000 as default is ok) 

Even with constant, it would be better to declare it once in the header.

Reply:
Added `CL2_LARGE_CLUSTER_NODES` parameter